_posts/2018-05-24-open-source-hackillinois-2018.markdown

```diff
@@ -0,0 +1,45 @@
+
+![Students start to work with mentor Andy Schwartzmeyer on the Apache Mesos project](/assets/article_images/open-source-hackillinois-2018/powershell.jpg "Apache Mesos")
+
+Last year, hacker Trent Spice worked with the Rust team and [won the Long Con prize]({{ site.baseurl }}{% post_url 2017-09-29-open-source-at-hackillinois-rust-cookbook %}) for HackIllinois 2017. This year, he came back with other students from Indiana University–Purdue University Indianapolis (IUPUI) and continued to work on Rust with mentors Steve Klabnik and Ashley Williams. HackIllinois aims to build these long-term relationships between students and the Open Source community, and we're super glad that groups from HackIllinois are still working together to contribute to and build on projects like [Rust Cookbook](https://github.com/rust-lang-nursery/rust-cookbook) and [RustBridge](https://github.com/rustbridge).
+
+![Hackers from IUPUI work with mentors from Rust in ECEB](/assets/article_images/open-source-hackillinois-2018/rust.jpg "Rust")
+
+We also had some awesome mentors representing Open Source projects from the fields of Data Science. We had one of the original founding members of [Julia](https://github.com/JuliaLang/julia), Jeff Bezanson, join us to show students the world of technical computing and had mentor Carol Willing (former Director of the Python Software Foundation) returning from HackIllinois 2017 to work on Jupyter.
+
+# Projects
+
+Tons of Open Source projects found themselves welcoming new contributors at HackIllinois. We had representatives from large Open Source projects such as [Apache Mesos](https://github.com/apache/mesos), [NPM](https://github.com/npm/npm), [Rust](https://github.com/rust-lang/rust), [Conda](https://github.com/conda/conda), [Homebrew](https://github.com/Homebrew/brew), and [Hyper](https://github.com/zeit/hyper). These awesome, influential projects gave students and mentors the opportunity to work together to contribute to some of the most common tools they use every day.
+
+HackIllinois was home to lots of smaller, growing Open Source projects, too, like [OpenTreeMap](https://github.com/OpenTreeMap/otm-core), [NodeSchool](https://github.com/nodeschool/nodeschool.github.io) and [PrairieLearn](https://github.com/PrairieLearn/PrairieLearn).  These Open Source projects have immense opportunities for growth at HackIllinois. In fact, mentor Michael Maurizi mentioned that he and his team solved every issue that he had set aside for HackIllinois 2018!
+
+HackIllinois did not limit students to only contributing to existing Open Source projects, though. Many students developed their own Open Source projects to give back to the community with their own new ideas. One project, [Puppeteer to Istanbul](https://github.com/istanbuljs/puppeteer-to-istanbul), even has 70+ stars on GitHub as of the time of publication and was [mentioned in a talk at Google I/O](https://youtu.be/lhZOFUY1weo?t=19m4s). And it was born right here at HackIllinois.
```

Comment:
could maybe link more projects than just this one (similar to the discussion on slack -- don't know what the conclusion from this was) not sure how feasible this is